haiku-apps/beshare/beshare-2.31.recipe

```diff
@@ -13,16 +13,19 @@ have to click \"refresh\" or anything first)
 - BeShare supports any number of simultaneous uploads and downloads \
 in parallel, but will serialize simultaneous file requests to any \
 given host for efficiency."
+HOMEPAGE="https://github.com/HaikuArchives/BeShare"
 COPYRIGHT="1999-2012 Jeremy Friesner"
 LICENSE="
 	Public Domain
 	libsanta
 	"
-HOMEPAGE="https://github.com/HaikuArchives/BeShare"
-SOURCE_URI="$(HOMEPAGE)/archive/b721fcc.tar.gz"
+SOURCE_URI="${HOMEPAGE}/archive/b721fcc.tar.gz"
```

Comment:
Exact. How did I not notice that before?!!!
So shall we just change this to:
```
srcGitRev="b721fcc4a4297b32738994b9c427d7db26c2c8fc"
SOURCE_URI="$HOMEPAGE/archive/$srcGirRev.tar.gz"
```
(which is the srcGitRev already in the recipe, in its shortened version), and leave 2.32 for a separate commit? (by you or anyone else)
Comments welcome, as usual, from all contributors ;-)